strr-api/tests/unit/resources/test_registration_applications.py

```diff
@@ -134,8 +134,7 @@ def test_get_applications_by_registration_type(session, client, jwt):
 
     assert HTTPStatus.OK == rv.status_code
     response_json = rv.json
-    # This will exclude Draft applications
-    assert len(response_json.get("applications")) == 3
+    assert len(response_json.get("applications")) == 4
```

Comment:
Is this also part of the issue? We should not be adding anything beyond the actual scope of the bug.